orchestra/utils/task_lifecycle.py

```diff
@@ -1031,6 +1031,17 @@ def update_related_assignment_status(task, assignment_counter, data,
         start_datetime=submit_datetime)
 
 
+def _call_abort_completion_function(project):
+    abort_completion_fn = project.workflow_version.abort_completion_function
+    abort_completion_fn_path = abort_completion_fn.get('path', '')
+    abort_completion_fn_kwargs = abort_completion_fn.get('kwargs', {})
+    if abort_completion_fn_path:
+        abort_completion_function = locate(abort_completion_fn_path)
+        abort_completion_function(
+            project.id,
```

Comment:
looking at some of the other functions we have in Orchestra, we pass in the task/step/etc. rather than the ID.
```suggestion
            project,
```